cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -418,17 +427,15 @@ def __init__(self, *component_gates: cirq.Gate) -> None:
             else:
                 self.component_gates += (gate,)
 
-    def qubit_index_to_gate_and_index(  # pylint: disable=missing-function-docstring
-        self, index: int
-    ) -> Tuple[cirq.Gate, int]:
+    def _qubit_index_to_gate_and_index(self, index: int) -> Tuple[cirq.Gate, int]:
         for gate in self.component_gates:
             if gate.num_qubits() > index >= 0:
                 return gate, index
             index -= gate.num_qubits()
         raise ValueError("index out of range")
 
     def qubit_index_to_equivalence_group_key(self, index: int) -> int:
```

Comment:
Shouldn't this have a docstring?

Reply:
Yes, I don't know why linter is not catching this.

Reply:
Oh! The docstring is already inherited from https://quantumai.google/reference/python/cirq/InterchangeableQubitsGate

Reply:
imo it should still be overwritten to something like "Returns a key that differs between qubits in different component gates and non-interchangeable qubits in the same component gate."

Reply:
Done.